pool.go

```diff
@@ -261,6 +273,7 @@ func NewPool(network, addr string, size int, opts ...PoolOpt) (*Pool, error) {
 		PoolPipelineConcurrency(size),
 		// NOTE if 150us is changed the benchmarks need to be updated too
 		PoolPipelineWindow(150*time.Microsecond, 0),
+		PoolWithTrace(trace.PoolTrace{}),
```

Comment:
Same as with the `WithClusterTrace` call: This is unnecessary, since a new Pool will already have `pt` set to an empty `PoolTrace`.

Reply:
I'll fix it.